addons/remove-search-bar-autocomplete/userscript.js

```diff
@@ -0,0 +1,8 @@
+export default async function ({ addon, global, console }) {
```

Comment:
Enable prettier at https://github.com/BroJac5246/ScratchAddons/actions

Reply:
I _think_ I got it? Can you check real quick? <sub>Sorry, like I said, I'm new to this.</sup>

Reply:
> I _think_ I got it? Can you check real quick? Sorry, like I said, I'm new to this.

Yes you got it